### DISCUSSION TITLE
Auto start a specific GBA game?

### DISCUSSION THREAD

**Discussion by jasongrieves** [profi200/open_agb_firm#136] [Ideas]
I've got it setup nicely to launch open_agb_firm automatically on boot and would love to automatically launch a game.  Maybe a config for automatically launching last game?  User can use a key combo to get out of that or just change the config when they want to not do that

**Comment by profi200** [comment]
autoboot.txt ~~See README~~.

edit:
Nevermind. This is actually not documented.
Place "autoboot.txt" in "/3ds/open_agb_firm" and add a path in the form of "sdmc:/my_roms_folder/my_game.gba". Obviously adjust the path to where your actual game is.
